windows-utils/README.txt

```diff
@@ -0,0 +1,8 @@
+In the occassion you are running a windows platform, we will have to make a few fixes for your machine to be able to build this app:
```

Comment:
I think @fhd probably meant the standard `README.md`

Reply:
It's fine here by me for now - I thought it's coolest if this is described in the top level README and if it's part of `build.js`, but we can always shuffle things around later. For now I'm kinda happy if it works :D

Reply:
My point is that it's probably better as a markdown file, as the rest of the directories...